pkg/server/server.go

```diff
@@ -2,59 +2,58 @@
 package server
 
 import (
-	"encoding/json"
 	"fmt"
+	"github.com/cmsgov/easi-app/pkg/handlers"
+	"github.com/okta/okta-jwt-verifier-golang"
 	"log"
 	"net/http"
-	"time"
+	"os"
+	"strings"
 )
 
-type status string
-
-const (
-	draft status = "DRAFT"
-)
+func isAuthenticated(authHeader string) bool {
+	tokenParts := strings.Split(authHeader, "Bearer ")
+	bearerToken := tokenParts[1]
+	if bearerToken == "" {
+		return false
+	}
 
-type profile struct {
-	Name         string
-	SystemOwners []string
-	CreatedAt    time.Time
-	UpdatedAt    time.Time
-	Status       status
-}
+	toValidate := map[string]string{}
+	toValidate["cid"] = os.Getenv("OKTA_CLIENT_ID")
+	toValidate["aud"] = "EASi"
 
-// HandleLanding is the handler for the root path
-func HandleLanding(w http.ResponseWriter, r *http.Request) {
-	sampleProfile := profile{
-		Name:         "My Favorite Project",
-		SystemOwners: []string{"admin@example.com", "operations@example.com"},
-		CreatedAt:    time.Now(),
-		UpdatedAt:    time.Now(),
-		Status:       draft,
-	}
-	js, err := json.Marshal(sampleProfile)
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
+	jwtVerifierSetup := jwtverifier.JwtVerifier{
+		Issuer:           os.Getenv("OKTA_ISSUER"),
+		ClaimsToValidate: toValidate,
 	}
 
-	w.Header().Set("Content-Type", "application/json")
-	enableCors(&w)
+	verifier := jwtVerifierSetup.New()
+
+	_, err := verifier.VerifyAccessToken(bearerToken)
```

Comment:
Regarding the last comment, this would then become the meat of the middleware during a request.